posts/ja/2023-02-18-23.0.0.1_ja.adoc

```diff
@@ -0,0 +1,199 @@
+// Contact/Reviewer: ReeceNana,cbridgha
+// // // // // // // // 
+[#eplv2]
+== Open LibertyのライセンスはEPLv2になりました
+2022年12月7日、ink:{url-prefix}/blog/2022/12/07/relicense-eplv2-proposal.html[Open LibertyのEPLv2への再ライセンス提案]のブログ記事を公開し、Open LibertyのライセンスをEPLv1からEPLv2への変更についてコミュニティに意見を求めました。 この提案に対して否定的なフィードバックがなかったため、私たちはこの変更を進めました。結果として、Open Liberty 23.0.0.1はEPLv2ライセンスのもとでリリースされます。 EPLv1からEPLv2への移行の理由と影響についてのより詳しい情報は、link:{url-prefix}/blog/2022/12/07/relicense-eplv2-proposal.html[proposal blog記事]をご覧ください。
+
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+
+[#bugs]
+== 主なバグ修正
+
+以下のセクションでは、このリリースで修正したバグの一部について説明します。興味がある場合はlink:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A23001+label%3A%22release+bug%22[full list of bugs fixed in 23.0.0.1]を参照してください。
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/23885[JDK20 以降では、最低でも 1.8 の jdkSourceLevel を使用]
++
+JDK 20 では、`javac` が Java 1.7 のコンパイルのサポートを終了しています。この変更により、JSPで `jdkSourceLevel="17"` (Java version 1.7) を指定した場合、以下のエラーが発生します。
++
+[source]
+----
+com.ibm.ws.jsp.JspCoreException: JSPG0049E: /xxxxxxxxx.jsp failed to compile :
+error: Source option 7 is no longer supported. Use 8 or later.
+----
++
+Java 20以降をランタイムとする場合、ユーザが指定した `jdkSourceLevel` に低いレベルが指定されると、 `18` に上書きされます。
+修正後、Java 20以降を使用する場合、`jdkSourceLevel` に `17` 以下が指定されると、その値は `18` に上書きされるようになりました。
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/23883[ファイルモニタリングでデフォルトのkeystore ファイルが検出されない]
++
+`updateTrigger` 属性に `mbean` (デフォルト値) を使用すると、変更が発生したときにデフォルトの keystore ファイルがリロードされないという問題が発生しました。
++
+この問題は解決され、このシナリオでファイル監視が適切に行われ、期待通りにファイルがリロードされるようになりました。
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/23782[JDBCDriverService; Booleanパラメータに関する問題]
++
+Microsoft JDBC ドライバの変更により、`properties.microsoft.sqlserver` プロパティの属性に boolean 値が使用された場合にリグレッションが発生するようになりました。この変更により、以下の例のようなエラーが発生します。
++
+[source]
+----
+com.ibm.ws.jdbc.internal.JDBCDriverService W DSRA8021W: Warning: error setting 'encrypt'=true: java.lang.IllegalArgumentException: argument type mismatch
+at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
+----
++
+この問題を解決するために、`boolean` 値を現在期待されている `String` に変換する変更が行なわれました。
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/23771[IndexOutOfBoundsExceptionがリソース停止中に発生することがある]
++
+バックエンドリソースの停止中に `java.lang.IndexOutOfBoundsException` が発生する可能性があります。停止中は、破棄するようにマークされた接続を再利用すべきではありません。不良な接続は削除されますが、次のマッチングによって再利用しようとすると、次のような例外がスローされる可能性があります。
++
+[source]
+----
+StackTrace=java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
+	at java.util.ArrayList.rangeCheck(ArrayList.java:670)
+	at java.util.ArrayList.remove(ArrayList.java:509)
+	at com.ibm.ejs.j2c.FreePool.getFreeConnection(FreePool.java:658)
+----
++
+この問題は解決され、`java.lang.IndexOutOfBoundsException` は発生しなくなりました。
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/23690[JTOpen Toolbox driver 11.1 Open LibertyからIBM iへのJDBC接続に失敗する]
++
+IBM JTOpen toolbox driver 11.1 を使用して IBM i システムとの JDBC 接続を確立すると、次のエラーで接続に失敗します。
```

Comment:
Should `toolbox driver` be capitalized? (If so, link above also needs `driver` to be capitalized)

Reply:
Made toolbox-> Toolbox.  
The driver stays in lowercase... [Greg's issue](https://github.com/OpenLiberty/open-liberty/issues/23690) and the [IBM doc](https://www.ibm.com/docs/en/was/8.5.5?topic=SSEQTP_8.5.5/com.ibm.websphere.nd.multiplatform.doc/ae/tcws_toolbox.htm) both seem to have it in lowercase.